net/dns/manager.go

```diff
@@ -331,6 +331,8 @@ func (m *Manager) NextPacket() ([]byte, error) {
 	return buf, nil
 }
```

Comment:
commit message:

`net/dns: time out DNS ...`

(timeout = noun, time out = verb, and DNS all caps in English)